### PR TITLE
Images for 8.0.0-beta.1

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Please be specific when describing the PR. Every detail helps: What are you changing? Why?
| Type?             | new feature
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | ~
| How to test?      | ~
| Possible impacts? | ~

### CODE REVIEW

**Review by yanmakouf — APPROVED** [review]

**Review by okom3pom — COMMENTED** [review]
_https://www.prestashop.com/download/old/prestashop_8.0.0-beta.1.zip_

404 for me

**Review by okom3pom — APPROVED** [review]

File: images/8.0.0-beta.1/7.2-apache/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM prestashop/base:7.2-apache
+LABEL maintainer="PrestaShop Core Team <coreteam@prestashop.com>"
+
+ENV PS_VERSION 8.0.0-beta.1
+
+# Get PrestaShop
+ADD https://www.prestashop.com/download/old/prestashop_8.0.0-beta.1.zip /tmp/prestashop.zip
```
Comment (atomiix) [inline images/8.0.0-beta.1/7.2-apache/Dockerfile:7]: The link is not the good one 😕

**Review by atomiix — CHANGES_REQUESTED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by okom3pom** [comment]
oh i just check https://github.com/PrestaShop/docker/commit/6f55564031f5e6524ba50122fc74134c3268a48b after my comment; sorry for my bad approve.

**Comment by hugofintecture** [comment]
Hello! Does this can be tested now? I don't see the new builds on Docker Hub https://hub.docker.com/r/prestashop/prestashop/tags?page=1&ordering=last_updated
Thanks in advance :)

**Comment by atomiix** [comment]
@hugofintecture it should be good now! https://hub.docker.com/layers/prestashop/prestashop/prestashop/8.0.0-beta.1/images/sha256-777ac9be3cbc8bcb2a3dc19d1176dce074e13aab7ab2d358a5c77236024aaaf4?context=explore

**Comment by hugofintecture** [comment]
Thank you for your reactivity! I'll try it :)

**Pull request merged**